alarm_table_defs.hpp

```diff
@@ -60,8 +76,6 @@ class AlarmTableDef
     _alarm_definition(alarm_definition),
     _severity_details(severity_details) {}
 
-  unsigned int state();
```

Comment:
Why have you removed this? I'm slightly surprised it still compiles without this line - have you run `make` and `make test` to check that?